README.md

```diff
@@ -58,14 +57,24 @@ node_templates:
     image:
         type: im.nodes.Image
         properties:
-            name: Centos-/
+            name: Centos-7
             config: 
                 id: 'https://fedcloud-services.egi.cesga.es:11443/51'
                 storage: 1024M
-                username: ...
-                password: ...
-                public_key: 'ssh-rsa ...'
-                private_key: |
+                username: ... # Optional
+                password: ... # Optional
+            use_external_resource: true
+            resource_id: default_image
+            simulate: False
+
+    keypair:
+        type: im.nodes.Keypair
+        properties:
+            name: Credentials
+            config: 
+                id: 'user@laptop'
+                public_key: 'ssh-rsa ...' # Optional
```

Comment:
Instead of optional, I understand that both keys doesn't have to be provided if `use_existing_resource: False`, and are mandatory if `use_existing_resource: True`.

Is that right? If so, please modify the docs.

Reply:
Then remove `use_existing_resource`, because it seems that it is implemented.

Also, the generation of the keys if they are not defined should be reflected somehow in the README.

simulate property is implemented?

Reply:
Yes, simulate is implemented. If it's `True` it does nothing during the operation